src/silo/database.cpp

```diff
@@ -273,18 +273,18 @@ void silo::Database::finalizeBuild() {
                max_count = count;
             }
          }
-         if (max_symbol == GENOME_SYMBOL::A || max_symbol == GENOME_SYMBOL::C || max_symbol == GENOME_SYMBOL::G || max_symbol == GENOME_SYMBOL::T || max_symbol == GENOME_SYMBOL::N) {
-            database_partition.seq_store.positions[partition_index].flipped_bitmap_for_symbol =
+         if (max_symbol == NUCLEOTIDE_SYMBOL::A || max_symbol == NUCLEOTIDE_SYMBOL::C || max_symbol == NUCLEOTIDE_SYMBOL::G || max_symbol == NUCLEOTIDE_SYMBOL::T || max_symbol == NUCLEOTIDE_SYMBOL::N) {
```

Comment:
Not for this PR, but I wonder why clang format doesn't break this quite long line...